sites-enabled/40-cdn.freecodecamp.org.conf

```
@@ -0,0 +1,32 @@
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+  server_name cdn.freecodecamp.org;
+  access_log /var/log/nginx/access.log combined if=$log_4xx_5xx;
+  # SSL TODO: add that conf file when it exists.
+  include snippets/common/ssl-cdn-org.conf;
+
+  # security
+  include snippets/common/security.conf;
+  location / {
+    add_header Access-Control-Allow-Origin *;
+    add_header Access-Control-Allow-Methods "GET,OPTIONS";
+    proxy_set_header Host freecodecamp-cdn.s3.amazonaws.com;
+    proxy_pass https://s3.amazonaws.com;
+  }
+}
+
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name cdn.freecodecamp.org;
+
+  return 301 https://cdn.freecodecamp.org$request_uri;
+}
+
+# health check
+server {
+  listen 64555;
+  return 200;
+}
```

```suggestion
}

```

Nitpick: Insert newline here. NGINX configs are packaged into a giant single file.